locale/fi/pagination.json

```diff
@@ -0,0 +1,62 @@
+{
+  "title": "Sivutus (Pagination)",
+  "intro": "Sivutus on komponentti, jonka avulla käyttäjä voi navigoida useille sivuille jaettua sisältöä. Sivustusta käytetään esimerkiksi hakutulosten näyttämiseen.",
+  "note.title": "Saavutettavuus ja käytettävyys",
+  "note.items": [
+    "Tieto sivun vaihdosta välitetään ruudunlukijalle. Komponentissa on visuaalisesti piilotettu kenttä tätä tarkoitusta varten.",
+    "Mikäli sivuja on paljon, käyttäjälle voidaan tarjota mahdollisuus sivunumeron syöttämiseen erilliseen sivunumerokenttään.",
+    "Komponentti ilmaisee tiedon sivun vaihtumisesta myös ruudunlukijakäyttäjille visuaalisesti piilotettua tekstisisältöä käyttäen.",
+    "Ruudunlukijalle sivunumeron tietokentän informaatio kannattaa antaa puhtaasti tekstimuodossa, esimerkiksi \"Sivu 1 kautta 20\", sillä erikoismerkkien lukutapa saattaa vaihdella.",
+    ""
+  ],
+  "sections": [
+    {
+      "title": "Mistä komponentti koostuu",
+      "paragraphs": [
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": "Sivutus koostuu kahdesta nuolipainikkeesa, tietokentästä sekä valinnaisesta sivunumeron syötekentästä. Nuolinpainikkeilla käyttäjä voi siirtyä yhden sivun kerrallaan joko eteen tai taaksepäin. Tietokenttä on nuolipainikkeiden välissä ja sen tarkoitus on kertoa käyttäjälle nykyisen sekä viimeisen sivun numerot. Syötekentän avulla käyttäjä voi siirtyä suoraan haluamalleen sivulle."
```

Comment:
Viiva puuttuu: "eteen ja taaksepäin" --> "eteen- ja taaksepäin"

Reply:
Korjattu